frontend/src/slices/chats/chats.slice.ts

```diff
@@ -66,7 +66,13 @@ const { reducer, actions, name } = createSlice({
     });
 
     builder.addCase(getCurrentChatMessages.fulfilled, (state, action) => {
-      state.currentChatMessages = action.payload.items;
+      state.currentChatMessages = {};
+
+      action.payload.items.forEach((item) => {
+        const date = new Date(item.createdAt).toDateString();
+        (state.currentChatMessages[date] =
+          state.currentChatMessages[date] ?? []).push(item);
+      });
```

Comment:
Grouping should be on backend side

Reply:
@v0ldemar01 Should it be in the chat messages sevice?

Reply:
It can be

Reply:
Done